pkg/constants/constants.go

```diff
@@ -0,0 +1,6 @@
+package constants
+
+const (
+	JWT_DOMAIN_SEPARATION_LABEL   = "jwt|"
+	PAYER_DOMAIN_SEPARATION_LABEL = "payer|"
```

Comment:
I'm going to add domain separation for originator envelopes as well in a separate PR

Reply:
Wondering if we can make these shorter, seeing as they're on every signature (which is included 2x per envelope)? And would this make sense as 1 or more bytes rather than a string, seeing as it's concatenated to bytes?

Reply:
The size kinda comes free, since it just gets added before the inputs are hashed and doesn't go into the signature itself.

Reply:
> And would this make sense as 1 or more bytes rather than a string, seeing as it's concatenated to bytes?

Go [doesn't let you use byte arrays as constants](https://blog.boot.dev/golang/golang-constant-maps-slices/). The alternative would be to define a function like `GetPayerDomainSeparationLabel` and have it return the value as a byte array. But I don't think that's any cleaner than this.